src/main/core/API/Serializer/Log/SecurityLogSerializer.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Claroline\CoreBundle\API\Serializer\Log;
+
+use Claroline\AppBundle\API\Options;
+use Claroline\AppBundle\API\Serializer\SerializerTrait;
+use Claroline\CoreBundle\API\Serializer\User\UserSerializer;
+use Claroline\CoreBundle\Entity\Log\SecurityLog;
+use Claroline\CoreBundle\Library\Normalizer\DateNormalizer;
+
+class SecurityLogSerializer
+{
+    use SerializerTrait;
+
+    private $userSerializer;
+
+    public function __construct(UserSerializer $userSerializer)
+    {
+        $this->userSerializer = $userSerializer;
+    }
+
+    public function serialize(SecurityLog $logSecurity): array
```

Comment:
$logSecurity
```suggestion
    public function serialize(SecurityLog $securityLog): array
```

(don't forget to update references below)

Reply:
Done, thx :)